instance/api/openedx_appserver.py

```diff
@@ -98,5 +98,5 @@ def make_active(self, request, pk):
             return Response(
                 {"error": "Cannot make an unhealthy app server active."}, status=status.HTTP_400_BAD_REQUEST
             )
-        app_server.instance.set_appserver_active(app_server.pk)
-        return Response({'status': 'App server updated.'})
+        set_appserver_active(app_server.pk)
+        return Response({'status': 'App server activation initiated.'})
```

Comment:
@antoviaque 

> What does "immaterial" mean? Even if it was not causing an issue (but are we sure this is the case?), I'd rather not leave errors like this in the logs - otherwise we'll just grow used to see traceback there, and start missing issues.

I've seen quite a few times that installing a package via pip shows tracebacks during the installation, without the installation failing. (E.g. pylint has done this for a while, but several other packages as well.)  I've looked at these cases in the past only to learn that this is normal, and as long as the installation succeeded it has been fine.  What's more, in these cases there was nothing that could be done to prevent the traceback from being shown.  Since pip succeeded in this case, I assumed that's one of these cases again.

The error is gone now.  Yesterday, we had a different error instead that actually did cause the installation to fail.  I've looked into it and fixed it.

Regarding the load balancer logs, the idea is that load balancers will eventually become managed by the instance manager itself.  Each instance will then link to the associated load balancer, and the load balancer will have all logs attached.  That's what I had in mind when I said "Since load balancers aren't shown in the UI yet, this means the messages aren't visible in the UI for now."

Your suggestions to add more logging at the instance level make sense.  They are not easy to implement, though.  If I add logging to the `get_load_balancer_configuration()` method, this will be executed everytime the load balancer gets reconfigured.  I'll think about this tomorrow and figure something out.